client/src/components/RestaurantsResults/RestaurantsResults.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router";
+import { getRestaurants } from "../../utils/serverCalls";
+
+import {
+  CardStyled,
+  Img,
+  Body,
+  Title,
+  P,
+  Details,
+  Price,
+  Rating,
+  Menu,
+  Directions,
+} from "./RestaurantsResultsStyles";
+
+export interface RestaurantData {
+  name: string;
+  place_id: string;
+  geometry: {
+    location: {
+      lat: number;
+      lng: number;
+    };
+  };
+  vicinity: string;
+  formatted_phone_number: string;
+  price_level: number;
+  rating: number;
+  url: string;
+  website: string;
+  opening_hours: {
+    weekday_text: [string];
+  };
+  photos: [
+    {
+      height: number;
+      html_attributions: [string];
+      photo_reference: string;
+      width: number;
+    }
+  ];
```

Comment:
This `photos` object needs to be a separate interface like so: 
```
export interface RestaurantPhotos {
   height: number;
   html_attributions: string;
   photo_reference: string;
   width: number;
}
```
Then inside of this `RestaurantData` interface, you can list the `photos` object like so:
```
photos: [RestaurantPhoto];
```